packages/components/Avatar/src/Avatar.types.ts

```diff
@@ -57,9 +57,6 @@ export const AvatarColors = [
   'mink',
   'platinum',
   'anchor',
-  'burgundy',
-  'hotPink',
-  'orchid',
```

Comment:
Why are these colors removed?

Reply:
These three shouldn't be part of the set of colors used in avatar color calculation. The full set of colors used by Fluent UI Apple ([link here](https://github.com/microsoft/fluentui-apple/blob/dfa5ede5894c3a3db5367b7a1c7965eaa99cfb8e/ios/FluentUI/Avatar/AvatarTokens.swift#L145-L176)):
```
darkRed
cranberry
red
pumpkin
peach
marigold
gold
brass
brown
forest
seafoam
darkGreen
lightTeal
teal
steel
blue
royalBlue
cornflower
navy
lavender
purple
grape
lilac
pink
magenta
plum
beige
mink
platinum
anchor
```

Reply:
win32 does use these colors though, and removing the colors from here affects that. Can you instead change the hash to only look at the first N-3 colors in the list?

Reply:
Why is there a design difference between web/win32 and mobile here? The list of colors feels like it should be a globally consistent thing. Should we take that up with design to update one or the other?

Reply:
afaik web and ios were using the same colors. so i am confused as well.

Reply:
Following up conversation offline

Reply:
@Saadnajmi , @harrieshin  These three colors are used in PersonaCoin component